README.md

```diff
@@ -16,27 +16,124 @@ This code for sample application is intended for demonstration purposes only. It
 * kubectl is installed - https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 * eksctl is installed - https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html
 * jq is installed - https://jqlang.github.io/jq/download/
+* [Optional] If you plan to install the infrastructure resources using Terraform, terraform cli is required. https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli
 
 # EKS demo
 
-## Build the sample application images and push to ECR
+## Deploy via Terraform
```

Comment:
Can we change the order so that Terraform is after Shell Script steps? Reason being that Terraform is marked as optional but having it as the first method implies that it's the recommended method.

Reply:
ok